command/bridge/server/server.go

```diff
@@ -98,15 +97,15 @@ func runCommand(cmd *cobra.Command, _ []string) {
 	closeCh := make(chan struct{})
 
 	// Check if the client is already running
-	if cid, err := helper.GetBridgeChainID(); !errors.Is(err, helper.ErrExternalChainNotFound) {
-		if err != nil {
-			outputter.SetError(err)
-		} else if cid != "" {
-			outputter.SetError(fmt.Errorf("external chain already running: %s", cid))
-		}
-
-		return
-	}
+	// if cid, err := helper.GetBridgeChainID(); !errors.Is(err, helper.ErrExternalChainNotFound) {
+	// 	if err != nil {
+	// 		outputter.SetError(err)
+	// 	} else if cid != "" {
+	// 		outputter.SetError(fmt.Errorf("external chain already running: %s", cid))
+	// 	}
+
+	// 	return
+	// }
```

Comment:
We certainly don't want to leave this part commented out. Either remove it or change it if necessary. 

IMO we can live without that, but don't take that as granted and double-check if we need it for some reason.